### PR TITLE
Changed the way AC idle RPM bump is handled

### PR DESCRIPTION
**Pull request by pchmura4**
Changed the way AC idle RPM bump is handled. Before it was always adding fixed RPM value.
In my example, the A/C compressor is happy with 900 RPM, fully warmed-up engine idles at 600, so I'm adding 300 RPM to have 900 RPM.
But... before warming up I'm Idling at more than 1000 RPM, so adding 300 RPM results with 1300 RPM idle that is not necessary...
This commit changes that, so the value is actually not an adder, but a target RPM for the A/C compressor so it works efficiently behaving like OEM system.

### CODE REVIEW

File: firmware/integration/rusefi_config.txt [hunk 1]
```diff
@@ -1060,7 +1060,7 @@ bit tempBooleanForVerySpecialLogic
 	float turboSpeedSensorMultiplier;;"mult", 1, 0, 0, 7000, 3
 
 	Gpio[CAM_INPUTS_COUNT iterate] camInputsDebug;
-	int16_t acIdleRpmBump;Extra idle target speed when A/C is enabled. Some cars need the extra speed to keep the AC efficient while idling.;"RPM", 1, 0, 0, 1000, 0
+	int16_t acIdleRpmBump;Idle target speed when A/C is enabled. Some cars need the extra speed to keep the AC efficient while idling.;"RPM", 1, 0, 0, 2000, 0
```
Comment (rusefillc) [inline firmware/integration/rusefi_config.txt:1063]: "bump" in the method name is no longer appropriate

while renaming variable hopefully we would find not-zero default value which would need to be updated

**Review by rusefillc — CHANGES_REQUESTED** [review]

**Comment by rusefillc** [comment]
Please mention the change in CHANGELOG (as a breaking change I guess?)

**Comment by ElDominio** [comment]
This works by guaranteeing a minimum RPM when AC is on, but not changing the RPM if it is over the guaranteed minimum correct? So in cold idle, you don't get extra RPMs you don't need? That seems good

File: firmware/integration/rusefi_config.txt (hunk 1 above)
Comment (pchmura4) [reply firmware/integration/rusefi_config.txt]: Hi, I have changed the names, default value and CHANGELOG, just not sure how to add this to this pull request 😅
https://github.com/pchmura4/rusefi/commit/5a948f2ae6bfcdb6730518ff937599c21118e1ea

**Comment by pchmura4** [comment]
> This works by guaranteeing a minimum RPM when AC is on, but not changing the RPM if it is over the guaranteed minimum correct? So in cold idle, you don't get extra RPMs you don't need? That seems good

Correct, probably the idle % adder would need to be a curve, but in my case I dont' use it at all, as idle PID does it's job perfectly

**Comment by rusefillc** [comment]
@pchmura4 note compilation errors

**Comment by pchmura4** [comment]
now it builds, checked on my PC

**Comment by rusefillc** [comment]
Let me attempt https://github.com/rusefi/rusefi/pull/5629 since this PR definitely does not need to change 73 files

**Comment by pchmura4** [comment]
Sure, thanks

**Comment by rusefillc** [comment]
Fixed by https://github.com/rusefi/rusefi/pull/5629

**Comment by rusefillc** [comment]
Thank you for the contribution!

@pchmura4 do you have a build thread on the forum?

**Comment by pchmura4** [comment]
I will create one when I'll finish my project 😄

**Pull request closed without merging**